src/ducks/billsMatching/Linker/billsToOperation/helpers.spec.js

```diff
@@ -0,0 +1,168 @@
+import {
```

Comment:
Not necessary to review the whole file in this commit, it is basically a copy/paste from cozy-konnector-libs at this stage.